nixos/modules/services/misc/sssd.nix

```diff
@@ -42,11 +42,6 @@ in {
   };
   config = mkMerge [
     (mkIf cfg.enable {
-      assertions = singleton {
```

Comment:
This is probably not the right move here. We are removing the option because somewhere else we have a `foo -> nscd.enabled` condition. if `foo` isn't the falsey value you could still enable `sssd` while `nscd` would be allowed to be disabled. Is that really desired?

Reply:
I don't understand this commend. `sssd` had its own additional assertion to complain when nscd was not enabled - but without nscd, sssd is as broken as other NSS modules.

`services.sssd.enable` sets `system.nssModules` (as any other module providing NSS modules), and we have a more global assertion complaining about this in `nsswitch.nix`.

which is now h

Reply:
I further improved the error message in `nsswitch.nix`, to also show the generated path of nssModules, so this should be covered.